src/Symfony/Component/Serializer/SerializerInterface.php

```diff
@@ -36,7 +36,7 @@ public function serialize($data, $format, array $context = []);
      * @param string $type
      * @param string $format
      *
-     * @return object
+     * @return mixed
```

Comment:
`object` was too strict. The serializer could for example also return an array:
```php
$serializer->deserialize('[{"foo":"bar"}]', 'Acme\DummyClass[]', 'json')
```

Reply:
I'm changing it to object|array for consistency with DenormalizerInterface